package.json

```diff
@@ -99,8 +100,8 @@
     "eslint-config-prettier": "6.11.0",
     "eslint-formatter-friendly": "7.0.0",
     "eslint-plugin-import": "2.22.0",
-    "eslint-plugin-prettier-internal-rules": "file:scripts/tools/eslint-plugin-prettier-internal-rules",
     "eslint-plugin-jest": "23.20.0",
+    "eslint-plugin-prettier-internal-rules": "file:scripts/tools/eslint-plugin-prettier-internal-rules",
```

Comment:
This happened when I tried re-adding the latest update of @types/node to devDependencies, would be happy to take it back out if needed.

Reply:
It's on me, I named it `eslint-plugin-internal-rules` at first.